apis/apis.json

```diff
@@ -5218,4 +5219,4 @@
       ]
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
Remove trailing newline, as otherwise it'll show up when we regenerate projects (I think).